addons/payment_ogone/models/payment_transaction.py

```diff
@@ -83,7 +84,7 @@ def _get_specific_rendering_values(self, processing_values):
         }
         if self.tokenize:
             rendering_values.update({
-                'ALIAS': payment_utils.singularize_reference_prefix(prefix='ODOO-ALIAS'),
+                'ALIAS': f'ODOO-ALIAS-{str(uuid.uuid4())}',
```

Comment:
remove the `-`, allowing selection via double-click
```suggestion
                'ALIAS': f'ODOO-ALIAS-{uuid.uuid4().hex}',
```

Reply:
https://github.com/odoo/odoo/pull/115564